src/main/config/config_streamer.c

```diff
@@ -339,7 +340,20 @@ static int write_word(config_streamer_t *c, config_streamer_buffer_align_type_t
     if (c->err != 0) {
         return c->err;
     }
-#if defined(STM32H7)
+#if defined(EEPROM_IN_RAM)
+    if (c->address == (uintptr_t)&eepromData[0]) {
+        memset(eepromData, 0, sizeof(eepromData));
+    }
+
+    uint64_t *dest_addr = (uint64_t *)c->address;
```

Comment:
`memcpy` would do just fine

Reply:
yes, as noted in #8379 an API needs to be extracted from the 5 implementations of config storage, this can be addressed then.